### PR TITLE
docs(website): add label package doc page

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Added label package doc page
- [x] Removed label page from cypress ignore list

### CODE REVIEW

**Review by nkrantz — APPROVED** [review]
Looks great! Just a reminder to update the airtable before merging

File: packages/paste-website/src/pages/components/label/index.mdx [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+<content>
+
+<LivePreview scope={{Input, Label, HelpText}} language="jsx">
+  {`<>
+  <Label htmlFor="email_address" required>Email address</Label>
+  <Input aria-describedby="email_help_text" id="email_address" name="email_address" type="email" placeholder="example@twilio.com" onChange="" required/>
+  <HelpText id="email_help_text">Please enter a valid email.</HelpText>
+</>`}
+</LivePreview>
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/pages/components/label/index.mdx:91]: I think we're going to start working it into being the norm. Simon suggested I add onto the skeleton  loader docs page, so doing the same here.

File: packages/paste-website/src/pages/components/label/index.mdx [hunk 2]
```diff
@@ -0,0 +1,354 @@
+---
+title: Label - Components
+package: '@twilio-paste/label'
+description: Label used to name Paste form fields.
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/label/index.mdx:4]: My only feedback is that we should come up with a better package description for here and in the `package.json` file. 

Let's get the team to have input on it and get consensus first, before we push up an update though. We can do it in the `plz` slack channel on this review item

Comment (richbachman) [reply packages/paste-website/src/pages/components/label/index.mdx:4]: Sounds good. Thanks for kicking that off.

**Review by gloriliale — APPROVED** [review]
This is great! I had the exact same question as Nora about the example coming above guidelines, so I'd like to hear more about what we're trying to achieve with that change, but this doc page is solid 👊

File: packages/paste-website/src/pages/components/label/index.mdx (hunk 2 above)
Comment (richbachman) [reply packages/paste-website/src/pages/components/label/index.mdx]: Updated.

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**